repose-aggregator/commons/utilities/src/test/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapperTest.scala

```diff
@@ -93,6 +94,7 @@ class HttpServletRequestWrapperTest extends FunSpec with BeforeAndAfter with Mat
       wrappedRequest.getHeaderNames.asScala.toList should contain theSameElementsAs headerMap.keys.filterNot( _ == "foo")
     }
 
+    //todo: fix this test -- it should not be case sensitive
```

Comment:
Sure enough, getting rid of equalsIgnoreCase is super easy. Rather than:
removedHeaders.filterNot(_.equalsIgnoreCase(headerName))

We could use:
removedHeaders - headerName